frontend/src/store/user.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+import _ from 'lodash';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { API_URL } from '../utils/apiConfig';
+
+export const userSlice = createSlice({
+  name: 'user',
+  initialState: {},
+  reducers: {
+    setUser: (state, { payload }) => {
+      return _.omit(payload, '__v')
+    }
+  }
+});
+
+export const { setUser } = userSlice.actions;
+export const userSelector = (store) => store.user;
+export default userSlice.reducer;
```

Comment:
You need to store your data in the local storage to avoid logging out your user after refreshing the page.